src/PidManagerFactory.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-expressive-swoole for the canonical source repository
+ * @copyright Copyright (c) 2018 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   https://github.com/zendframework/zend-expressive-swoole/blob/master/LICENSE.md New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Zend\Expressive\Swoole;
+
+use Psr\Container\ContainerInterface;
+use Psr\Log\LoggerInterface;
+
+class PidManagerFactory
+{
+
+    public function __invoke(ContainerInterface $container) : PidManager
+    {
+        $pidFile = '/tmp/zend-swoole.pid';
```

Comment:
This should likely be `sys_get_temp_dir() . '/zend-swoole.pid`, OR `/var/run/zend-swoole.pid`. Better: allow it to be configured, with a sane default if the configuration is not present. This will allow users to have multiple Swoole servers running on different ports without collision.